app/src/main/java/org/simple/clinic/newentry/ButtonState.kt

```diff
@@ -0,0 +1,6 @@
+package org.simple.clinic.newentry
+
+enum class ButtonState {
+  CLICKED,
+  ENABLED
+}
```

Comment:
Please add a line feed here.

You can enable the setting in Android Studi / Idea to automatically add a line feed on save.
`Preferences > Editor > General > Ensure line feed at file end on Save`